environment.yml

```diff
@@ -4,53 +4,53 @@ channels:
 dependencies:
```

Comment:
so these are the generic deps

Reply:
This file should have all the dependencies for development excluding the specific to the backends. The section of soft dependencies is probably not very accurate right now (I think couple should be in the backends). But will have a look at those dependencies later.

Reply:
can you add this comment at the top